docs/feature_debounce_type.md

```diff
@@ -33,7 +33,9 @@ The debounce code is compatible with split keyboards.
 # Changing between included debouncing methods
 You can either use your own code, by including your own debounce.c, or switch to another included one.
 Included debounce methods are:
-* eager_pk - debouncing per key. On any state change, response is immediate, followed by ```DEBOUNCE_DELAY``` millseconds of no further input for that key
+* eager_pr - debouncing per row. On any state change, response is immediate, followed by locking the row ```DEBOUNCE_DELAY``` milliseconds of no further input for that key. 
+For use in keyboards where refreshing ```NUM_KEYS``` 8-bit counters is computationally expensive / low scan rate, and fingers usually only hit one row at a time.
+* eager_pk - debouncing per key. On any state change, response is immediate, followed by ```DEBOUNCE_DELAY``` milliseconds of no further input for that key
```

Comment:
I have one question: This does not make it fully clear to the casual reader if this is a good algo to try on an ErgoDox or not. Would it be possible to make this explicit in the docs?

Reply:
Maybe adding a note that this is good for boards like the Ergodox EZ, which have a rotated matrix, where you can (usually) only hit one key in a "row" at a time (since the matrix row is actually a column on the keyboard).

Reply:
Updated, explaining the 90° rotation results in "rows" being columns and fingers normally being in a single column in normal use.